content/en/real_user_monitoring/platform/connect_rum_and_traces.md

```diff
@@ -553,6 +553,15 @@ Datadog uses the distributed tracing protocol and sets up the HTTP headers below
 
 Example:
 : `traceparent: 00-00000000000000008448eb211c80319c-b7ad6b7169203331s-01`
+
+`tracestate: dd=s:[sampling priority];o:[origin]`
+: `dd`: Datadog's vendor prefix.
+: `sampling priority`: Always set to `1` to make sure that the Agent keeps the trace.
```

Comment:
<!-- Questions are appropriate if you have a potential concern but are not quite sure if it's relevant or not. Asking the author for clarification or investigation can lead to a quick resolution.  -->
**❓ question:** ‏ Does it not represent if the trace is sampled or not? (`1` or `0`)